DragaliaAPI/Services/Game/StoryService.cs

```diff
@@ -134,25 +134,33 @@ int storyId
     private async Task<IEnumerable<AtgenBuildEventRewardEntityList>> ReadUnitStory(int storyId)
     {
         UnitStory data = MasterAsset.UnitStory[storyId];
-
+        StoryData charaStory = MasterAsset.CharaStories[data.ReleaseTriggerId];
+        List<AtgenBuildEventRewardEntityList> rewardList = new();
         DbPlayerStoryState story = await storyRepository.GetOrCreateStory(data.Type, storyId);
-
         int wyrmiteReward;
+        int emblemRewardId;
+        Entity emblemRewardEntity;
```

Comment:
why do these need to be declared out here?

Reply:
Force of habit, I'm used to declaring all my variables at the beginning of the function, would you rather have me declare them only in the path where I use them ?

Reply:
On second thought, it's only a style preference -- fine to leave as is